modules/redpanda/redpanda.go

```diff
@@ -11,7 +11,6 @@ import (
 
 	"github.com/docker/go-connections/nat"
 
-	"github.com/testcontainers/testcontainers-go"
```

Comment:
I believe this import must be kept here 😄

Reply:
Oh my goland did something unexpected :) will fix it in the next push